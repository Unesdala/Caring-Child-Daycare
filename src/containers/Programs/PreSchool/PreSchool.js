import React, { Component } from 'react';
import commonUtils from '../../../components/lib/commonUtils';
import DefaultPreSchoolContent from './PreSchoolContent';
import DefaultFooter from '../../../components/Footer/Footer';

export class PreSchool extends Component {
  constructor(props) {
    super(props);
    this.commonUtils = commonUtils;
  }

  componentDidMount() { this.commonUtils.setTitleAndScroll('Twos'); }

  render() {
    return (
      <div className="page-content">
        <DefaultPreSchoolContent />
        <DefaultFooter />
      </div>
    );
  }
}

export default PreSchool;
