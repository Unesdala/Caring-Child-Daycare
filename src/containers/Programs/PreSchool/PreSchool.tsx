import React, { Component } from 'react';
import CommonUtils from '../../../lib/commonUtils';
import DefaultPreSchoolContent from './PreSchoolContent';

export class PreSchool extends Component {
  public commonUtils: typeof CommonUtils = CommonUtils;

  componentDidMount(): void { this.commonUtils.setTitleAndScroll('Pre School', window.screen.width); }

  render(): JSX.Element {
    return (
      <>
        <div className="page-content">
          <DefaultPreSchoolContent />
        </div>
      </>
    );
  }
}

export default PreSchool;
