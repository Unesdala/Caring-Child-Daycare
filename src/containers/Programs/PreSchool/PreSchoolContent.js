import React from 'react';

const PreSchoolContent = () => (
  <main className="pre-school">
    <p />
  </main>
);

export default PreSchoolContent;
