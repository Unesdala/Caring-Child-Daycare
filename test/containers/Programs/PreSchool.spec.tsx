/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { PreSchool } from 'src/containers/Programs/PreSchool/PreSchool';
import PreSchoolContent from 'src/containers/Programs/PreSchool/PreSchoolContent';

describe('PreSchool', () => {
  const wrapper = shallow(<PreSchool />);
  it('renders', () => {
    expect(wrapper.find(PreSchoolContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(PreSchoolContent).dive().find('main.pre-school-main--content').exists()).toBe(true);
  });
  
});
