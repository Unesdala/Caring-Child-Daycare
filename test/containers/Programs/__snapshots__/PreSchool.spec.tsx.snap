// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreSchool renders 1`] = `ShallowWrapper {}`;
